### ISSUE TITLE
after feature branch merge devel compile gives errors

### ISSUE THREAD

**Issue by ppxasjsm** [michellab/Sire#150]
Having merged the feature branch, I tried to do a clean compile of devel. (Only made changes that should not affect the actual running of the simulation so far.)
When trying to run I get the following error and am a bit puzzled as to what may be going wrong here:

```
Traceback (most recent call last):
  File "/home/ppxasjsm/dev_sire.app/pkgs/sire-2016.3.1/share/Sire/scripts/somd-freenrg.py", line 146, in <module>
    OpenMMMD.runFreeNrg(params)
  File "/home/ppxasjsm/dev_sire.app/lib/python3.6/site-packages/Sire/Tools/__init__.py", line 172, in inner
    retval = func()
  File "/home/ppxasjsm/dev_sire.app/lib/python3.6/site-packages/Sire/Tools/OpenMMMD.py", line 1562, in runFreeNrg
    moves = setupMovesFreeEnergy(system, ranseed, gpu.val, lambda_val.val)
  File "/home/ppxasjsm/dev_sire.app/lib/python3.6/site-packages/Sire/Tools/OpenMMMD.py", line 1132, in setupMovesFreeEnergy
    Integrator_OpenMM = OpenMMFrEnergyST(molecules, solute, solute_hard, solute_todummy, solute_fromdummy)
Boost.Python.ArgumentError: Python argument types in
    OpenMMFrEnergyST.__init__(OpenMMFrEnergyST, MoleculeGroup, MoleculeGroup, MoleculeGroup, MoleculeGroup, MoleculeGroup)
did not match C++ signature:
    __init__(_object*)
```

Is this still an issue with python 3.6 and OpenMM compatibility? I am a bit lost as to which version I should be working on right now, to make sure gradients are output using the right units and somd/somd-freenrg actually runs. 
Compilation was just done using ./compile_sire.sh on my usual Ubuntu machine.

**Comment by chryswoods** [comment]
That error happens when Sire has been compiled without OpenMM support. Was OpenMM properly installed and detected during compilation with compile_sire.sh?

**Comment by chryswoods** [comment]
Just to say that I have updated the merged analyse_freenrg.py so that it works properly for analysing freenrgs.s3 files. There were a few changes needed, i.e. moving imports to your mbar section so that they are not unnecessarily imported when not using mbar, fixing some parsing as this had become lost somehow, and also some extra fixes I've made that fixed bugs that were outstanding on my end.

**Comment by ppxasjsm** [comment]
Mmmh I am not sure I did anything differently in terms of OpenMM support than before. As in I could run compile_sire.sh before and everything was detected and worked fine out of the box. 

As for analyse_freenrg.py I tried to not really mess with any of the original script, so I hope it wasn't difficult to find any issues. I guess I had only tried to run one gradient and one s3 file that both seemed to give the right result.
